docs/admin_api/user_admin_api.rst

```diff
@@ -1,3 +1,47 @@
+List Accounts
+=============
+
+This API returns all local user accounts.
+
+The api is::
+
+    GET /_synapse/admin/v2/users?from=0&limit=10&guests=false
+
+including an ``access_token`` of a server admin.
+The parameters ``from`` and ``limit`` are required only for pagination.
+By default, a ``limit`` of 100 is used.
+The parameter ``user_id`` can be used to filter by user id using an SQL regexp.
+The parameter ``guests=false`` can be used to exclude guest users,
+default is to include guest users.
+The parameter ``deactivated=true`` can be used to include deactivated users,
+default is to exclude deactivated users.
+If the endpoint does not return a ``next_token`` then there are no more users left.
+It returns a JSON body like the following:
+
+.. code:: json
+
+    {
+        "users": [
+            {
+                "user_id": "<user_id1>",
```

Comment:
are you sure it returns `user_id` rather than `name` here? I can't see anywhere in the code that it would return user_id.

Reply:
Ah right, I didn't look into the implementation of `_simple_search_list_txn` in detail. So I'm basically missing the wrapping `%`s to get the desired result...